frontend/packages/kubevirt-plugin/src/selectors/vm/devices.ts

```diff
@@ -1,5 +1,32 @@
+import { VMKind, V1NetworkInterface, VMLikeEntityKind } from '../../types';
+import { DEVICE_TYPE_DISK, DEVICE_TYPE_INTERFACE } from '../../constants';
+import { V1Disk } from '../../types/vm/disk/V1Disk';
+import { getDisks, getInterfaces } from './selectors';
+import { asVM } from './vmlike';
+
+type BootableDeviceType = {
+  type: string;
+  value: V1Disk | V1NetworkInterface;
+};
```

Comment:
nitpick: I was told that we should declare all types at the bottom of the file. I'm not sure if this is actually an established convention, though.

Reply:
good point :-) I am stacking other work on top of this and moving the type to another file completely

Reply:
Ah! Carry on, then. =) I didn't notice the WIP in the title. Sorry for the added noise.